source/main/ipc/get-logs.js

```diff
@@ -25,7 +25,20 @@ export const getLogsChannel: MainIpcChannel<
   GetLogsMainResponse
 > = new MainIpcChannel(GET_LOGS_CHANNEL);
 
-const isFileAllowed = (fileName: string) => includes(ALLOWED_LOGS, fileName);
+const isFileAllowed = (fileName: string) => {
+  let allowed = includes(ALLOWED_LOGS, fileName);
```

Comment:
This part is hard to understand if you don't know the context. Please, change to the following:

```js
const isFileAllowed = (fileName: string) =>
  includes(ALLOWED_LOGS, fileName) || isOldDaedalusLog(fileName);
```

Then:

```js
const isOldDaedalusLog = (fileName: string) => {
  // add the checking here, with proper comments for easy understanding
}
```

Reply:
Great tip @daniloprates!